Privacy Policy

```diff
@@ -0,0 +1,27 @@
+Klooni1010 Privacy policy
+
+This Privacy Policy has been created in order to disclose our information collection, use and disclosure practices for the application Kolooni! 10 10 puzzle game.
+Kolooni! 10 10 puzzle game is an entertainment application, that allows users arrange bricks of different sizes and color shapes.
```

Comment:
Name is not consistent. Above you use `Klooni1010` and here `Klooni 1010`, while the correct one is in my opinion `1010! Klooni`.

Reply:
That's not what a typo is.